test_unstructured_ingest/expected-structured-output/confluence-diff/MFS/1540126.json

```diff
@@ -335,7 +335,7 @@
         "eng"
       ]
     },
-    "text": "\\uD83D\\uDD17 Reference materials",
+    "text": "\\uD83D\\uDD17 Reference materials",
```

Comment:
Is this just different whitespace?

Reply:
The old one is a non-breaking space and the new one is a plain space.

This is an aspect of the whitespace-normalization difference. Each run of whitespace in the text for an element is reduced to a single space (after leading and trailing whitespace are stripped from the paragraph).

`&nbsp;` is included in whitespace as are `"\n"`, `"\t"`, and several others.